src/Compilers/VisualBasic/Portable/Syntax/SyntaxKind.vb

```diff
@@ -3174,6 +3174,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' </summary>
         EndOfInterpolatedStringToken = 787                              ' PunctuationSyntax : SyntaxToken
 
+        ''' <summary>
+        ''' Represents a single keyword in a VB program. Which keyword can be determined
+        ''' from the Kind property.
+        ''' </summary>
+        LoadKeyword = 788                                               ' LoadKeyword : SyntaxToken
+
+        ''' <summary>
+        ''' Represents a #load directive appearing in scripts.
+        ''' </summary>
+        LoadDirectiveTrivia = 789                                       ' LoadDirectiveTrivia : DirectiveTriviaSyntax : StructuredTriviaSyntax
```

Comment:
I would have thought we'd have left gaps near the other keywords and trivia, but that doesn't seem to be the case.  I guess we just add to the end in VB?